### ISSUE TITLE
Does CVE-2019-7653 affect v4.2.2 or only Debian-specific package?

### ISSUE THREAD

**Issue by jameswettenhall** [RDFLib/rdflib#902]
Version 4.2.2 of rdflib is listed as insecure in https://github.com/pyupio/safety-db/blob/master/data/insecure_full.json

If it was just a Debian-specific vulnerability which has been patched in the Debian package, are there any plans to notify the Python Safety DB maintainers that rdflib 4.2.2 is still safe to use?

**Comment by randomstuff** [comment]
Yes it's a a problem specirfic to the Debian package due to the way the CLI programs are implemented:

```sh
#!/bin/sh

exec /usr/bin/python -m rdflib.tools.rdf2dot $*
```

When installing from pip you get this instead:

```python
#!/home/foo/my-venv/bin/python3
# -*- coding: utf-8 -*-
import re
import sys

from rdflib.tools.rdf2dot import main

if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
    sys.exit(main())
```

Debian derivatives are probably impacted as well (Ubuntu is impacted).

**Comment by rillian** [comment]
Given safety-db is ignoring the fix for this, could you please publish a 4.2.3 package so users have a way of resolving this CVE in reports?

**Comment by gjhiggins** [comment]
Timed out
